content/en/docs/tasks/debug/debug-cluster/_index.md

```diff
@@ -313,7 +313,16 @@ This is an incomplete list of things that could go wrong, and how to adjust your
 - Action: applications (containers) designed to tolerate unexpected restarts
   - Mitigates: Node shutdown
   - Mitigates: Kubelet software fault
+  
+### Things to note after dockershim migration
 
+## {{< note >}} 
+*Note: Due to change in CNI, the `kubelet` binary might not recognize `--network-plugin` flag even after passing to `kubelet` which leads to the following error:
```

Comment:
IMO, this could be phrased better. WDYT?
```suggestion
*Note: Starting 1.24, since management of the CNI is no longer in scope for kubelet, the `--network-plugin` command line parameter has been removed from the binary and will throw the following error, if used:
```

Reply:
As per @reylejano suggested [here](https://github.com/kubernetes/website/pull/34764#discussion_r984924005), I've omitted this line.